src/components/MultiSelect/MultiSelect.tsx

```diff
@@ -392,7 +393,7 @@ export const MultiSelect: ChakraComponent<
                 layout="column"
                 isFullWidth
                 isRequired={false}
-                labelText="Multi select checkbox group label"
+                labelText={buttonText}
```

Comment:
I feel that the button label text, on its own, is not descriptive enough here.  It simply mirrors what is already shown in the button.  What about adding `options` with it?

Ex: MultiSelect options

Reply:
Hmm yes, I added just the `buttonText` based on what Clare said but I'll ask for a review and ask if adding "options" is better.

Reply:
@bigfishdesign13 talked with Clare and she noted that the button text alone is fine, if anyone passes "Format options" as the button text, then we'll have a redundant legend text as "Format options options".